metadata/demographic/unpopulated_areas_2010_approx.md

```diff
@@ -0,0 +1,68 @@
+# Title
+
+Utah Unpopulated Areas 2010 Approx
+
+## ID
+
+87112b87-1e64-49e7-a291-e35e188ca250
+
+## Brief Summary
+
+Polygon dataset of unpopulated areas in Utah dissolved as a single feature.
+
+## Summary
+
+This dataset contains a polygon representing unpopulated areas in Utah as of the 2010 Census. For 2020 Census data products, please see the [Demographic Data Index](https://gis.utah.gov/products/sgid/demographic/) available from the SGID.
+
+## Description
+
+### What is the dataset?
+
+Census blocks are small geographic zones through which population data can be collected and organized. These blocks are often bounded by features such as roads, rivers, or property lines. This dataset includes Census blocks in unpopulated areas of Utah as of the 2010 Census that have been dissolved into a single polygon.
```

Comment:
Let's de-emphasize the census block mentions due to uncertainty of data creation.  I think the PopBlockAreas2010_Approx was probably used to help create this, but we should focus more in this data and less on the blocks.
```suggestion
This dataset is a representation of Utah areas that have little to no population, buildings, or addresses, dissolved into a single polygon.
```